src/rest/orders.rs

```diff
@@ -57,13 +57,14 @@ pub struct PlaceActiveOrderData {
     pub order_type: OrderType,
     pub price: Option<f64>,
     pub time_in_force: TimeInForce,
-    pub close_on_trigger: Option<bool>,
+    pub close_on_trigger: bool,
     pub order_link_id: Option<OrderLinkId>,
     pub take_profit: Option<f64>,
     pub stop_loss: Option<f64>,
     pub tp_trigger_by: Option<TriggerPrice>,
     pub sl_trigger_by: Option<TriggerPrice>,
-    pub reduce_only: Option<bool>,
+    pub reduce_only: bool,
+    pub position_idx: Option<i32>,
```

Comment:
Could we make this an `Option<i64>` instead?

Reply:
yes but the linear order requires it as mandatory. I will revert this and the other non-optional